atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -472,8 +472,8 @@ public static SerializableTransactionManager createForTest(
                 metricsManager,
                 keyValueService,
                 new LegacyTimelockService(timestampService, lockService, lockClient),
-                NoOpLockWatchManager.INSTANCE,
-                NoOpLockWatchEventCache.INSTANCE,
+                NoOpLockWatchManager.create(),
+                NoOpLockWatchEventCache.create(),
```

Comment:
Strictly speaking, this is a bit weird (the LWM has a different cache to the one passed directly below), but it's for tests. No strong pref if you think we should resolve this.